functions/lib/clients/twitterClient.js

```diff
@@ -0,0 +1,58 @@
+const twitterClient = require('twitter')
+
+class Twitter {
+  constructor(consumer_key, consumer_secret, access_token, access_token_secret) {
+    this.client = new twitterClient({
+      consumer_key: consumer_key,
+      consumer_secret: consumer_secret,
+      access_token_key: access_token,
+      access_token_secret: access_token_secret
+    });
+  }
+
+  tweetMedia(mediaSize, mediaType, mediaData) {
+    return this._initUpload(mediaSize, mediaType)
+      .then((data) => {
```

Comment:
Changed this to mediaId (it's the mediaId string that identifies the uploaded file on Twitter).